climakitae/explore/warming.py

```diff
@@ -329,3 +331,69 @@ def _anom_allowed(self):
         else:
             self.param["anom"].objects = ["Yes", "No"]
             self.anom = "Yes"
+
+
+def drop_invalid_wrf_sims(ds):
```

Comment:
This could definitely be dropped in favor of the `_drop_invalid_sims` function.

Reply:
Will leave for now until we figure out how to integrate the existing warming levels code with this new stuff